adam-cli/src/main/scala/org/bdgenomics/adam/cli/TransformAlignments.scala

```diff
@@ -131,6 +131,10 @@ class TransformAlignmentsArgs extends Args4jBase with ADAMSaveAnyArgs with Parqu
   var storageLevel: String = "MEMORY_ONLY"
   @Args4jOption(required = false, name = "-disable_pg", usage = "Disable writing a new @PG line.")
   var disableProcessingStep = false
+  @Args4jOption(required = false, name = "-partition_by_start_pos", usage = "EXPERIMENTAL: Save the data partitioned by genomic range bins based on start pos using Hive-style partitioning.")
```

Comment:
I would vote for dropping EXPERIMENTAL from these, one could argue that everything in ADAM is experimental until we hit version 1.0 and start semantic versioning (#338, #123).